css-color/Overview.bs

```diff
@@ -691,7 +758,7 @@ The ''currentcolor'' keyword</h3>
 	</div>
 
 	Note: Multi-word keywords in CSS usually separate their component words with hyphens.
-	''currentcolor'' doesn't, because it was originally introduced in SVG
+	''<color>/currentcolor'' doesn't, because it was originally introduced in SVG
 	as a special attribute value spelled "currentColor",
```

Comment:
No, it was introduced as a property value (and properties in SVG can be expressed in presentational attributes, as well as in style attributes, style elements, and external stylesheets).